include/mtx/public_rooms_chunk.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
```

Comment:
Put this inside the public_rooms header, since there is really no need to split the dependent types up.